airflow/dags/create_external_tables/rt_service_alerts_outcomes.yml

```diff
@@ -1,11 +1,6 @@
 operator: operators.ExternalTable
 bucket: gs://rt-parsed
 prefix_bucket: true
-
```

Comment:
They fail with the new required partition filter, but they don't really get noticed unless external tables are being run. Since this PR was adding two new ones, the issue popped up.